Chrysalis-Papilio/Soen-490-Papilio-Android
# Papilio 100 user account page UI

# General info

**Issue number**: 100

**Task description**: 

 - changes to the UI of the account menu 
 - small changes to the UI of the user profile

# Testing
To test this feature you have to:

1. open the app 
2. log in to your account 
3. go to the account tap (the 4th button of the nav bar)
4. verify that the new account menu is visible and usable 
5. click on the profile button 
6. verify that the small changes are displayed on the page
